multi-node/aws/README.md

```diff
@@ -5,24 +5,86 @@ View the full instructions at https://coreos.com/kubernetes/docs/latest/kubernet
 
 ## Development
 
+### Download pre-built binary
+
+```sh
+wget https://<binary-url>
+# check checksum
+chmod +x ./kube-aws
+sudo mv kube-aws /usr/bin/
+```
+
 ### Build
 
 Run the `./build` script to compile `kube-aws` locally.
-This depends on having golang available on your workstation.
+
+This depends on having:
+* golang >= 1.5
+
 The compiled binary will be available at `./bin/kube-aws`.
 
-### Custom Kubernetes Manifests
+## Initialize an asset directory
+```sh
+$ mkdir my-cluster
+$ cd ./my-cluster
+$ kube-aws init --cluster-name=my-cluster-name --external-dns-name=my-cluster-endpoint --region=us-west-1 --availability-zone=us-west-1c --key-name=key-pair-name
+```
 
-You may deploy a cluster using a custom CloudFormation template, Kubernetes manifests and install scripts using the `artifactURL` option in your cluster config.
+There will now be a ./cluster.yaml file in the asset directory.
 
-For example, you might upload a modified set of manifests to a custom S3 bucket (making the files publicly-readable) using the following commands:
+## Render contents of the asset directory
 
+```sh
+$ kube-aws render
 ```
-$ kube-aws render --output=artifacts/template.json
-$ aws s3 cp --recursive --acl=public-read artifacts/ s3://<bucket>/
+This generates the default set of cluster assets in your asset directory. These assets are templates and credentials that are used to create, update and interact with your Kubernetes cluster.
+
+You can now customize your cluster by editing asset files:
+
+* **./cluster.yaml**
```

Comment:
nit: same as above, would suggest dropping the "./" prefixes. Here and in headers below